src/promptflow-devkit/promptflow/_cli/_pf/_flow.py

```diff
@@ -595,6 +599,7 @@ def serve_flow(args):
         host=args.host,
         port=args.port,
         skip_open_browser=args.skip_open_browser,
+        engine=args.engine,
```

Comment:
mock the real start app function to assert engine passed correctly? may refer to test_flow_serve_cli.py::test_flow_serve